docs/contributing/style-guide.md

```diff
@@ -135,10 +136,10 @@ one-line installer requires fewer steps than manually installing from source."
 ### Avoid slang, metaphors, and jargon
 
 A particular word, phrase, or metaphor you're familiar with might not translate well to the other cultures featured
-among Netdata's global community. It's recommended you avoid slang or colloquialisms in your writing.
+among Netdata's global community. We recommended you avoid slang or colloquialisms in your writing.
 
-If you must use industry jargon, such as "white-box monitoring," in a document, be sure to define the term as clearly
-and concisely as you can.
+If you must use industry jargon, such as "white-box monitoring," in a document, define the term as clearly and concisely
+as you can.
 
```

Comment:
Let's add something like this (as a section even): No usage of abbreviations that have not been defined before. I may be the first one to break this rule but it should still be in the rules...

Reply:
We can move the abbreviation section up here maybe. Let me know